rectools/visuals/visual_app.py

```diff
@@ -357,7 +608,7 @@ class VisualApp(VisualAppBase):
 
     def __init__(
         self,
-        recos: TablesDict,
+        recos: tp.Union[pd.DataFrame, tp.Dict[tp.Hashable, pd.DataFrame]],
```

Comment:
```suggestion
        recos: tp.Union[pd.DataFrame, TablesDict],
```

Reply:
This one is on purpose. I don't really want to confuse users with TablesDict. So I left `tp.Dict[tp.Hashable, pd.DataFrame]` in each class which is directly used by users.

Reply:
hmm, we didn't do this before

but ok, if you think this helps..

Reply:
Explicit better then implicit here I guess.